### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by acabal** [AdmiralAsshat/arthur-conan-doyle_the-valley-of-fear#1]
CSS

- We can remove the single use `inscription` class. Instead, target that blockquote with `#chapter-1-1 blockquote:first-of-type`.

- Additionally that blockquote does not need spans separating the lines. That pattern is only used for poetry/verse. p + br without spans is fine in this situation.

Content.opf

- Please add `<meta property="se:collection">Sherlock Holmes</meta>` after line 42

- For transcriber metadata, please create two separate entries, one for Toby and one for Al. Al can have the additional MARC relator of `mrk` for markup.

- In the long-description, use `<i>` instead of `<em>` for italics on books. Other semantics are not necessary in the long-description.

Other

- It looks like you used `<em>` without semantics for all instances of italics. `<em>` stands for emphasis, which means something spoken in an emphasized tone of voice. Things like book titles are not emphasis, they are plain italics, `<i>`. Additionally plain `<i>` tags need semantics to indicate what they are. For example `<em>La Jeune Fille à l’Agneau</em>` should be `<i epub:type="se:name.visual-art.painting">La Jeune Fille à l’Agneau</i>`. Note that letters spoken as letters are just `<i>` without semantics, see the typography manual.

- Chapter 1-1 line 57, there's a stray br at the end of the p tag.

- `se lint` throws an error about a missing nbsp char in the colophon

**Comment by acabal** [comment]
Also missing transcriber-sort in content.opf.

**Comment by AdmiralAsshat** [comment]
Hi Alex,

I believe I have fixed everything you brought up.

Somehow I introduced something that made the linter unhappy, and I'm not sure what.
```[xml|attr] selector in CSS, but no XML namespace declared (@namespace xml "http://www.w3.org/XML/1998/namespace";). ```

I can fix it by adding `@namespace xml "http://www.w3.org/XML/1998/namespace";` to the local.css, but I've found only [one](https://github.com/standardebooks/victor-hugo_les-miserables_isabel-f-hapgood/blob/43b14ec7382bc8ccd9a29226af5da8c72f2ca897/src/epub/css/local.css) SE repo where that namespace exists, so I'm guessing this is not usually necessary.

I've held off on committing until I hear from you.
